src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -59,68 +66,112 @@ class PaymentInformation extends React.Component {
     }
 
     const { paymentInformation } = this.props;
+    const directDepositNotSetup = !get(
+      'responses[0].paymentAccount.accountNumber',
+      paymentInformation,
+    );
 
-    // @todo Determine what an uninitialized state really looks like -
-    // Is responses null? Do we really need to check responses.length?
-    // Is there a paymentAccount, but containing only empty values?
-
-    if (!paymentInformation.responses || !paymentInformation.responses.length) {
-      return null;
+    let content = null;
+
+    if (paymentInformation.error) {
+      content = <LoadFail information="payment" />;
+    } else if (!this.props.profile.multifactor) {
+      content = <PaymentInformation2FARequired />;
+    } else if (directDepositNotSetup) {
+      content = (
+        <button
+          className="usa-button vads-u-margin-top--2"
+          onClick={this.editModalToggled}
+        >
+          Add your direct deposit information
+        </button>
+      );
+    } else {
+      const paymentAccount = paymentInformation.responses[0].paymentAccount;
+
+      content = (
+        <>
+          <div className="vet360-profile-field">
+            <ProfileFieldHeading onEditClick={this.props.editModalToggled}>
+              Bank name
+            </ProfileFieldHeading>
+            {paymentAccount.financialInstitutionName}
+          </div>
+          <div className="vet360-profile-field">
+            <ProfileFieldHeading onEditClick={this.props.editModalToggled}>
+              Account number
+            </ProfileFieldHeading>
+            {paymentAccount.accountNumber}
+          </div>
+          <div className="vet360-profile-field">
+            <ProfileFieldHeading onEditClick={this.props.editModalToggled}>
+              Account type
+            </ProfileFieldHeading>
+            {paymentAccount.accountType}
+          </div>
+          <p>
+            <strong>Note:</strong> If you think you’ve been the victim of bank
+            fraud, please call us at 800-827-1000 (TTY: 800-829-4833), and
+            select 5. We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m.
+          </p>
+          <PaymentInformationEditModal
+            onClose={this.props.editModalToggled}
+            onSubmit={this.props.savePaymentInformation}
+            isEditing={this.props.paymentInformationUiState.isEditing}
+            isSaving={this.props.paymentInformationUiState.isSaving}
+            fields={this.props.paymentInformationUiState.editModalForm}
+            editModalFieldChanged={this.props.editModalFieldChanged}
+            responseError={this.props.paymentInformationUiState.responseError}
+          />
+        </>
+      );
```

Comment:
Here's where all of the different state are handled.

1. If there's an error property in the `paymentInformation` data, then the fetch failed and we should show an error.
2. If `this.props.profile.multifactor` is `false`, then we should render a prompt for the user to enable 2FA.
3. If `responses[0].paymentAccount.accountNumber` is empty, then we should render a button prompting the user to set up direct deposit for the first time.
4. If none of that is true, then render the user's payment info